unified-runtime/source/adapters/level_zero/async_alloc.cpp

```diff
@@ -28,22 +28,22 @@ static ur_result_t enqueueUSMAllocHelper(
   std::scoped_lock<ur_shared_mutex> lock(Queue->Mutex);
 
   // Allocate USM memory
-  ur_usm_pool_handle_t USMPool = nullptr;
+  ur_usm_pool_handle_t UrPool = nullptr;
```

Comment:
just curious, why the name change?

Reply:
Since we use `UsmPool` as a wrapper for `EnqueuedPool` I thought it would be a good idea to be consistent and differentiate between different pool structures.